### PR TITLE
Update generators

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/rectangles/RectanglesTest.fs [hunk 1]
```diff
@@ -9,109 +9,69 @@ open Rectangles
...
+    rectangles strings |> should equal 0
 
 [<Fact(Skip = "Remove to run test")>]
 let ``One rectangle`` () =
-    let input = 
-      [ "+-+"; 
-        "| |"; 
-        "+-+" ]
-    rectangles input |> should equal 1
+    let strings = ["+-+"; "| |"; "+-+"]
```
Comment (robkeim) [inline exercises/rectangles/RectanglesTest.fs]: @ErikSchierboom is there a reason you stopped outputting these on multiple lines? I think they were visually easier to understand before this change.

Comment (robkeim) [reply exercises/rectangles/RectanglesTest.fs]: I think this line needs to be updated:
https://github.com/ErikSchierboom/fsharp/blob/18ab904e83809cb315d6298918a001a988d6743b/generators/Generators.fs#L1035

**Comment by ErikSchierboom** [comment]
@robkeim Good catch! I've fixed the issue.

**Comment by robkeim** [comment]
Thanks @ErikSchierboom! These changes are good to merge as soon as Travis is done building.

**Pull request merged**